src/apiCalls.js

```diff
@@ -0,0 +1,5 @@
+export const getMovies = async () => {
```

Comment:
This is the main change. I build the fetch using the async await syntax.

Reply:
Great first step for getting data